src/clusterfuzz/_internal/base/tasks.py

```diff
@@ -274,6 +274,8 @@ def get_postprocess_task():
   # This should only be run on non-preemptible bots.
   if not task_utils.is_remotely_executing_utasks():
     return None
+  if not environment.platform().lower() == 'linux':
```

Comment:
A comment here would be helpful to explain the intent behind the exclusion:

```suggestion
  # Postprocess is platform-agnostic, so we run all such tasks on our
  # most generic and plentiful bots.
  if not environment.platform().lower() == 'linux':
```